externs/shaka/manifest.js

```diff
@@ -76,6 +77,8 @@
  * @property {boolean} sequenceMode
  *   If true, we will append the media segments using sequence mode; that is to
  *   say, ignoring any timestamps inside the media files.
+ * @property {string} type
+ *   Indicates the type of the manifest. It can be HLS or DASH.
```

Comment:
At first it may sound weird that we need this, but it is necessary to prevent us from making any regressions in everything current, plus it opens the door to multiple improvements in HLS that I am working on in parallel.

Reply:
Done!